lib/modules/blockchain_process/blockchain.js

```diff
@@ -305,6 +305,12 @@ Blockchain.prototype.isClientInstalled = function (callback) {
     if (err || !stdout || stderr.indexOf("not found") >= 0 || stdout.indexOf("not found") >= 0) {
       return callback(__('Ethereum client bin not found:') + ' ' + this.client.getBinaryPath());
     }
+    const supported = this.client.isSupportedVersion(this.client.parseVersion(stdout));
```

Comment:
Can we have the parsed version be its own variable?

Reply:
Done.